### PR TITLE
Add all the langs

### PR DESCRIPTION
**Pull request by todd-dembrey**
@incuna/backend

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,3 +29,13 @@ translations-all:
...
+	@make translations locale=hu
+	@make translations locale=se
+	@make translations locale=lv
+	@make translations locale=lt
+	@make translations locale=cz
+	@make translations locale=es
+	@make translations locale=sl
+	@make translations locale=ee
+	@make translations locale=hr
+	@make translations locale=ro
```
Comment (meshy) [inline Makefile]: /me twitches in the general direction of the non-alphabetically-sorted list `:P`

Comment (adam-thomas) [inline Makefile]: Yeah but now you can pronounce it as 'Sleehro'

**Comment by adam-thomas** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (todd-dembrey) [inline Makefile]: @meshy Happy now?

**Pull request merged**